charts/lagoon-remote/templates/insights-remote.service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "lagoon-remote.insightsRemote.fullname" . }}
+  labels: {{- include "lagoon-remote.insightsRemote.labels" . | nindent 4 }}
```

Comment:
I would, out of habit, put anything nindented on the line it's to be added in (ie 4sp in, in the line below)